app/controllers/exercises_controller.rb

```diff
@@ -26,7 +26,7 @@ def new
 
   def create(id)
     kit_id = id
```

Comment:
Why we need this value (kit_id) if it use only once (move it to the method's incoming params)